src/Common/src/Interop/Windows/advapi32/Interop.RegistryOptions.cs

```diff
@@ -46,20 +46,5 @@ internal partial class RegistryOperations
             internal const int STANDARD_RIGHTS_READ = READ_CONTROL;
             internal const int STANDARD_RIGHTS_WRITE = READ_CONTROL;
         }
-
```

Comment:
I think it may be better to move this whole file to CoreLib, and not split it into a bunch of little files. All these constants are likely going to be used together. They are needed in CoreLib - they are under different names in CoreLib right now.